pkg/api/admin/openshiftcluster.go

```diff
@@ -130,14 +130,23 @@ const (
 	MTU3900 MTUSize = 3900
 )
 
+// OutboundType represents
```

Comment:
nit: a slightly more informative comment would be good

Reply:
+1